andela/travel_tool_front
# #161045193 Enable travel admin change requester room

#### What does this PR do?
Adds feature to enable travel admin change the room/bed of a requester

#### Description of Task to be completed?
During or prior to the active occupancy of a room, there may be reasons to move requesters to another rooms
A should be able to change the room of a requester so that he/she can ensure high satisfaction of the requester during his stay at the guest house

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-change-requester-room-161045193`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back)
- Navigate to `localhost:3000` in a browser
- Assign yourself the manager role and Travel Admin role so as to be able to approve your request in the database
- Navigate to the requests page and make a new request
- Navigate to the approvals page and approve your request
- Navigate to the residence manage page using the left navigation bar
- Click on a guesthouse to view it's details
- In the timeline component, you select your booking record and click change room
- Fill the form in the popup and submit
- You should get a notification stating the room was changed successfully

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161045193](https://www.pivotaltracker.com/story/show/161045193)
[#161045385](https://www.pivotaltracker.com/story/show/161045385)

#### Screenshots (if appropriate)
<img width="1431" alt="screen shot 2018-10-16 at 8 35 45 am" src="https://user-images.githubusercontent.com/13545380/46999859-c2aad600-d11e-11e8-9fe7-227793b96709.png">
<img width="1440" alt="screen shot 2018-10-18 at 8 05 35 pm" src="https://user-images.githubusercontent.com/13545380/47180635-e2a5e980-d318-11e8-9f8a-079719844494.png">

<img width="776" alt="screen shot 2018-10-16 at 8 36 00 am" src="https://user-images.githubusercontent.com/13545380/46999861-c2aad600-d11e-11e8-9f35-f32048e853b5.png">
<img width="1440" alt="screen shot 2018-10-18 at 8 04 22 pm" src="https://user-images.githubusercontent.com/13545380/47180652-ecc7e800-d318-11e8-8ef7-ebb2e3ac5a69.png">



#### Questions:
